backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -112,22 +109,24 @@ public BoardConfigDTO getJiraConfiguration(BoardType boardType, BoardRequestPara
 		try {
 			JiraBoardConfigDTO jiraBoardConfigDTO = getJiraBoardConfig(baseUrl, boardRequestParam.getBoardId(),
 					boardRequestParam.getToken());
-			CompletableFuture<List<TargetField>> targetFieldFuture = getTargetFieldAsync(baseUrl, boardRequestParam);
-			CompletableFuture<List<TargetField>> ignoredTargetFieldFuture = getIgnoredTargetFieldAsync(baseUrl,
-					boardRequestParam);
 			CompletableFuture<JiraColumnResult> jiraColumnsFuture = getJiraColumnsAsync(boardRequestParam, baseUrl,
 					jiraBoardConfigDTO);
-
-			return jiraColumnsFuture.thenCombine(targetFieldFuture,
-					(jiraColumnResult, targetFields) -> getUserAsync(boardType, baseUrl, boardRequestParam)
-						.thenApply(users -> BoardConfigDTO.builder()
-							.targetFields(targetFields)
-							.jiraColumnResponse(jiraColumnResult.getJiraColumnResponse())
-							.ignoredTargetFields(ignoredTargetFieldFuture.join())
-							.users(users)
-							.build())
-						.join())
-				.join();
+			CompletableFuture<List<TargetField>> targetFieldFuture = getTargetFieldAsync(baseUrl, boardRequestParam);
+			List<TargetField> ignoredTargetFields = getTargetFieldAsync(baseUrl, boardRequestParam).join()
+				.stream()
+				.filter(this::isIgnoredTargetField)
+				.toList();
+
+			return jiraColumnsFuture.thenCombine(targetFieldFuture, (jiraColumnResult, targetFields) -> getUserAsync(
+					boardType, baseUrl, boardRequestParam)
+				.thenApply(users -> BoardConfigDTO.builder()
+					.targetFields(
+							targetFields.stream().filter(targetField -> !isIgnoredTargetField(targetField)).toList())
```

Comment:
It's better to create a local variable to calculate this value as you did for ignoredTargetFields. So that it will be easier for maintenance.